src/identity/components/useVerified.ts

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useState } from 'react';
+import { base } from 'viem/chains';
+import type { Address } from 'viem';
+
+import { getEASAttestations } from '../getEASAttestations';
+
+export function useVerified(address: Address) {
+  const [verified, setVerified] = useState(false);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const attestation = await getEASAttestations(address, base);
```

Comment:
We can pass in the `options` to `getEASAttestations` here. Do we want to introduce an `IdentityProvider` to get the `options` to `getEASAttestations`?